src/components/Identity/index.js

```diff
@@ -37,6 +39,17 @@ const createIdentity = ({ config, logger, cookie }) => {
       },
       // Waiting for opt-in because we'll be reading the ECID from a cookie
       onBeforeEvent(event) {
+        const nowInHours = Math.round(
```

Comment:
This `nowInHours` guy seems like a good candidate for a util since you are using it in both modules.

Reply:
There's only one place where I use it.